cookbooks/munin/files/default/rrddump.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+RRD_DIR=/var/lib/munin/openstreetmap
+DIR=`mktemp -d`
+NPROCS=8
```

Comment:
I think I might be wrong about this - there was a problem with CPU but that was because munin only really uses one CPU so it was maxed out while all the others just sat there.

Also switching to CGI mode has reduced the amount it needs on each run.